web/backend/schemas/matches.ts

```diff
@@ -0,0 +1,21 @@
+import { Bson } from "../../../deps/web/deps.ts";
+import { db } from "../db/mongo.ts";
+
+// Schema for AYTO participants
+interface MatchesSchema {
+  _id: Bson.ObjectId;
+  matches: {
+    man: {
+      name: string;
+      pic: string;
+    };
+    woman: {
+      name: string;
+      pic: string;
+    };
+    probability: number;
+  };
+}
+
```

Comment:
Please create Schemas for MatchBox and Matching Nights

Reply:
already did
its on another branch :D